internal/scanner/scanner_test.go

```diff
@@ -846,7 +846,8 @@ func TestScanWithMTLS(t *testing.T) {
 	err = client.Get(context.TODO(), types.NamespacedName{Name: string(pod.GetUID()), Namespace: "namespace"}, &podPolicyReport)
 	require.NoError(t, err)
 
-	log.Debug().Any("podPolicyReport", podPolicyReport).Msg("podPolicyReport")
+	slog.Debug("podPolicyReport",
```

Comment:
I think we can safely remove this, it looks like the original line was a leftover.